a10_octavia/network/drivers/neutron/a10_octavia_neutron.py

```diff
@@ -134,14 +135,14 @@ def get_plugged_parent_port(self, vip):
     def create_port(self, network_id, subnet_id=None, fixed_ip=None):
         new_port = None
         if not subnet_id:
-            subnet_id = self.neutron_client.get_network(network_id).subnets[0]
-        fixed_ip = {'subnet_id': subnet_id}
+            subnet_id = self.get_network(network_id).subnets[0]
+        fixed_ip_dict = {'subnet_id': subnet_id}
```

Comment:
Let's change this to `fixed_ip_subnet`

Reply:
Removed this variable itself as in VRID code.